api/python/quilt3/data_transfer.py

```diff
@@ -437,11 +439,21 @@ def _copy_file_list_internal(file_list, message, callback):
     if not file_list:
         return []
 
-    total_size = sum(size for _, _, size in file_list)
+    results = _copy_file_list_internal_retry(file_list, [None] * len(file_list), message, callback)
```

Comment:
Can you simplify this by putting the tenacity retry logic around _copy_file_list_internal and setting reraise=True?

Reply:
I'm not quite sure what you exactly mean, but the results list should be preserved across retries, so I added parameter for it and helper function because I didn't want to change signature of `_copy_file_list_internal()`.